### DISCUSSION TITLE
V199 not detecting frames

### DISCUSSION THREAD

**Discussion by AnthonyZ60Hz** [collin80/SavvyCAN#565] [Q&A]
I'm using the EVTV Due with GVRET installed using GVRETupdate. Am able to connect with serial, but no frames populating. I've tried all the CAN speeds listed on the device datasheet (Elcon 3.3 kW charger). 
Any advice would be amazing.
![image](https://user-images.githubusercontent.com/122182466/211171791-f70140a9-ec59-4892-827f-242a93b6bc2f.png)

**Comment by collin80** [comment]
The console does show that communication is happening between SavvyCAN and the GVRET adapter so that's good. Have you independently confirmed that traffic is indeed flowing? Maybe there's a termination problem? Also, you can try bringing up a serial program and going to the COM port for the adapter directly. The speed for the elcon charger really should be 250k so in the serial console you could confirm this got set properly (use ? followed by the ENTER key to get a menu) and also see if you can see any traffic that way.

**Reply by AnthonyZ60Hz** [reply]
Thank you so much for replying Collin. I'll try and run through process of elimination for those possibilities.
